scss/style/_footer.scss

```diff
@@ -0,0 +1,59 @@
+$footer-text-color: $color-white;
+$footer-link-color: $color-white;
+$footer-social-color: $color-white;
+$footer-link-hover-color: $color-periwinkle;
+$footer-copyline-color: $color-silver;
+$footer-padding-lr: 30px;
+$footer-primary-color: $color-azure;
+$footer-secondary-color: $color-st-tropaz;
+$footer-bottom-border: $color-indigo;
+$footer-sides-offset: 50px;
+
+.clear-gutter {
```

Comment:
Watch out, this would be a utility and would need to go into the `utilities` folder. However, I'd add the margin reset to the correspnding classes directly, so the developer doesn't have to take care about the implementation specifics.